src/App.js

```diff
@@ -2,12 +2,61 @@ import React from 'react';
 
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRed } from './api/goods';
+import GoodsList from './components/GoodsList';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+class App extends React.Component {
+  state = {
+    goods: [],
+  }
+
+  getAllGoods = async() => {
+    const prepearedGoods = await getAll();
+
+    this.setState({ goods: prepearedGoods });
+  }
+
+  get5FirstGoods = async() => {
+    const prepearedGoods = await get5First();
+
+    this.setState({ goods: prepearedGoods });
+  }
+
+  getRedGoods = async() => {
+    const prepearedGoods = await getRed();
+
+    this.setState({ goods: prepearedGoods });
+  }
```

Comment:
these functions could be combined into one, which accepts a callback (getAll, get5First or getRed), executes it and write the result to state